tests/test_utilities.py

```diff
@@ -25,6 +25,19 @@ def test_valid_filenames():
         " Leading Space": "leading_space",
         "Trailing Space ": "trailing_space"
     }
+    print(console.green + "------------------ Assertions ------------------\n" + console.reset)
     for name, expected_name in names.items():
         print(console.cyan + repr(name) + ": " + console.yellow + expected_name + " [" + utilities.get_valid_filename(name) + "]" + console.reset)
-        assert(utilities.get_valid_filename(name) == expected_name)
\ No newline at end of file
+        assert(utilities.get_valid_filename(name) == expected_name)
+
+
+def test_get_fully_qualified_name():
```

Comment:
missing test for builtin type.

Reply:
question - why do we care about builtin types?

Reply:
I think just in case anyone passes in a builtin, it just strips the builtin string from it. I gather this is just from a beautification point of view, but might be because it's ambiguous on some platforms? 

I'm happy with stripping it just to keep builtin representations simple. If it solves other problems, great.

Reply:
I'm not sure what you mean by:
> but might be because it's ambiguous on some platforms?

Reply:
Just hazarding a guess of that 'other' reason. 

Regardless if there is a technical reason for it, I'd prefer the builtin class types to show up as `str` instead of `builtin.str` since that is less intuitive and typically hidden from the user (you never use `builtin.str` when programming).